core-web/apps/dotcms-ui/src/app/portlets/dot-edit-page/components/dot-favorite-page/dot-favorite-page.component.ts

```diff
@@ -165,6 +164,10 @@ export class DotFavoritePageComponent implements OnInit, OnDestroy {
                 } else {
                     this.form.get('thumbnail').setValue('');
                     this.store.setShowFavoriteEmptySkeleton(true);
+                    console.warn(
+                        'We apologize for the inconvenience. The following error occurred while generating a thumbnail for this page:',
+                        event?.detail?.error
+                    );
```

Comment:
Should we make this message `multi-language`? Something like:

```suggestion
                    console.warn(
                        this.dotMessagePipe.transform('favorite.pages.generate.thumbnail.error'),
                        event?.detail?.error
                    );
```

Reply:
This is just a suggestion. Feel free to ignore it If you don't think it is worthy it.

Reply:
test? not sure if we can do a spy on `console.warn`